src/Alert/Alert.component.tsx

```diff
@@ -0,0 +1,182 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import classNames from 'classnames';
+import styled, {
+    css,
+    FlattenInterpolation,
+    ThemeProps,
+} from '@xstyled/styled-components';
+import { th } from '@xstyled/system';
+import { transparentize, darken } from 'polished';
+// COMPONENTS
+import { Close } from '../Icon';
+import { Typography } from '../Typography';
+import { Button } from '../Button';
+import { get } from '../utils/get/get';
+
+export enum AlertTypes {
+    success = 'SUCCESS',
+    info = 'INFO',
+    warning = 'WARNING',
+    error = 'ERROR',
+}
+
+type AlertType =
+    | 'success'
+    | 'info'
+    | 'warning'
+    | 'error'
+    | AlertTypes.success
```

Comment:
Is there a particular raisin for doing both like this?

Reply:
> raisin

lol

I want to support TS enums as well as plains JS strings.